proto/tasks.proto

```diff
@@ -44,6 +49,16 @@ message TaskUpdate {
     google.protobuf.Timestamp now = 4;
 }
 
+// A task details update
+message TaskDetails {
+    // The task's ID which the details belong to.
+    common.SpanId task_id = 1;
+
+    google.protobuf.Timestamp now = 2;
+
+    Details details = 3;
```

Comment:
nit, take it or leave it: i'm not convinced this needs to be a separate message --- i think it would be fine just to put the histogram (and any other data we might add) in the `TaskDetails` message directly. should make the generated code a bit easier to work with that way...